github/repos_hooks.go

```diff
@@ -11,64 +11,27 @@ import (
 	"time"
 )
 
+// Deprecated: Please use PushEvent instead.
 // WebHookPayload represents the data that is received from GitHub when a push
 // event hook is triggered. The format of these payloads pre-date most of the
 // GitHub v3 API, so there are lots of minor incompatibilities with the types
 // defined in the rest of the API. Therefore, several types are duplicated
 // here to account for these differences.
 //
 // GitHub API docs: https://help.github.com/articles/post-receive-hooks
```

Comment:
It's not viable to place the "// Deprecated: [...]" comment at the top of a symbol's documentation, since that breaks the convention that an exported identifier named "Thing" should have documentation that begins with "Thing [...]". See https://go.dev/wiki/Deprecated. (It can be at the very bottom or in the middle, just not at the top.)

This applies here and in other places where the deprecation comment is added.

Reply:
Ah, yes... thank you, Dmitri!

Reply:
Thanks, marking resolved.